### ISSUE TITLE
About results ./scripts/etth2.sh

### ISSUE THREAD

**Issue by luossa** [DC-research/TEMPO#16]
Dear Author:
Thank you for sharing the code implementation of your work. I have been carefully studying your paper and codebase, but I have come across a discrepancy that I would like to clarify.

In your paper, in '5.1 ABLATION STUDY' section, you have discussed the 'Dec Loss' function, the results show that the 'Dec Loss' is useful for the final results. But when run   './scripts/etth2.sh', I find that the 'Dec Loss' is not calculated in total loss. 

the code in 'main_multi_6domain_release.py' line 327-329:

327    if args.model == 'GPT4TS_multi' or args.model == 'TEMPO_t5':
328                if not args.no_stl_loss:
329                     loss += args.stl_weight*loss_local
when run  "./scripts/etth2.sh", the args.model = "TEMPO", so the 'Dec Loss'  is not calculated in total loss.

And I run './scripts/etth2.sh' and './scripts/etth2_test.sh' try to follow your paper results, my reproducing results(pred_len = 192) are mse:0.3726384411119435 mae:0.3956169232930223, your paper results(pred_len = 192) are mse: 0.355 mae: 0.389. My reproducing results are approaching the results of your paper (pred_len = 336, mse=0.379  mae=0.408) . 
Is the hyperparameters incorrect？ 
Is the 'Dec Loss' affect my reproducing results lead to my reproducing results different from your paper results?

I appreciate your time and effort in addressing these queries, as they are critical to understanding and reproducing the results presented in your paper.

Thank you!

Best regards,
luossa
